tests/haml/HamlLintBearTest.py

```diff
@@ -0,0 +1,116 @@
+from queue import Queue
+
+from coalib.settings.Section import Section
+from coala_utils.ContextManagers import prepare_file
+from coalib.testing.LocalBearTestHelper import LocalBearTestHelper
+from coalib.testing.LocalBearTestHelper import verify_local_bear
+from coalib.testing.BearTestHelper import generate_skip_decorator
+from bears.haml.HamlLintBear import HamlLintBear
```

Comment:
A line to be left after `from coalib.testing.BearTestHelper import generate_skip_decorator` import.